Cabal/Cabal.cabal

```diff
@@ -459,3 +485,21 @@ test-suite package-tests
   ghc-options: -Wall -rtsopts
   default-extensions: CPP
   default-language: Haskell98
+
+test-suite parser-tests
+  if !flag(parsec-debug)
+    buildable: False
```

Comment:
`parsec-debug` is `default: False, manual: True`, so I'd be surprised if it was on by default.